spec/controllers/api_v1_slides_controller_spec.rb

```diff
@@ -1,13 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::SlidesController, type: :controller do
+  before(:each) { request.env["HTTP_ACCEPT"] = 'application/json' }
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->